apps/builddao/widget/components/modals/propose/AddMember.jsx

```diff
@@ -1,12 +1,20 @@
 const { Button } = VM.require("${config_account}/widget/components") || {
   Button: () => <></>,
 };
-
+const { InfoPopup } = VM.require(
+  "${config_account}/widget/components.modals.InfoAlert",
+) || {
+  InfoPopup: () => <></>,
+};
 const DaoSDK =
   VM.require("sdks.near/widget/SDKs.Sputnik.DaoSDK") || (() => <></>);
 
 const [accountId, setAccountId] = useState("");
 const [role, setRole] = useState("");
+const [infoPopup, setInfoPopup] = useState(false);
+const [copied, setCopied] = useState(false);
+const url =
+  "https://www.nearbuilders.org/buildhub.near/widget/app?page=feed&tab=proposals";
```

Comment:
let's update this
btw, why do we have this in two places?

Reply:
Yeah I will update this :)